packages/textfield/test/textfield.test.ts

```diff
@@ -342,6 +342,37 @@ describe('Textfield', () => {
             : null;
         expect(input).to.not.be.null;
     });
+    it('handles `name` attribute', async () => {
```

Comment:
Again, curious to see a test where `name` is not specified by the user.

Reply:
done!